gdnative-core/src/access.rs

```diff
@@ -34,6 +34,9 @@ where
 pub unsafe trait Guard: Drop {
     type Target;
     fn len(&self) -> usize;
+    fn is_empty(&self) -> bool {
+        self.len() == 0
+    }
```

Comment:
This one should not be added, because `Guard` is a trait for typed array access guards, not something that end users are expected to use directly. The reason why clippy suggests `is_empty` is for convenience. There are no conveniences to be had here.